.github/workflows/buildwheel.yml

```diff
@@ -155,3 +155,33 @@ jobs:
       - run: bin/install_flint_ubuntu.sh ${{ matrix.flint-tag }}
       - run: pip install .
       - run: python -m flint.test --verbose
+
+  # Deploy wheels and sdist to PyPI
+
+  pypi_release:
+    name: Publish to PyPI
+    needs: [build_wheels, build_sdist]
+    # Run only when a tag is pushed to the flintlib/python-flint repo
+    if: "github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && github.owner == 'flintlib'"
+    environment:
+      name: pypi
+      url: https://pypi.org/p/sympy
```

Comment:
Once this is merged I will test pushing a tag to trigger release.

Reply:
I might need a few attempts while fixing up the Actions workflow.

I guess it might work first time...